src/n-dhcp4.h

```diff
@@ -63,6 +61,7 @@ enum {
         N_DHCP4_CLIENT_EVENT_EXTENDED,
         N_DHCP4_CLIENT_EVENT_EXPIRED,
         N_DHCP4_CLIENT_EVENT_CANCELLED,
+        N_DHCP4_CLIENT_EVENT_LOG,
```

Comment:
The new event must be mentioned in n_dhcp4_client_pop_event() documentation.

Reply:
done